src/System.IO.Pipelines/System.IO.Pipelines.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\System.Buffers.Primitives\System.Buffers.Primitives.csproj" />
-    <ProjectReference Include="..\System.Buffers.Experimental\System.Buffers.Experimental.csproj" />
+    <ProjectReference Include="..\System.Collections.Sequences\System.Collections.Sequences.csproj" />
```

Comment:
Buffers.Experimental indirectly pulled in the Collections.Sequences project. Pipelines depended on both. We are breaking the dependency on Experimental, so I had to add an explicit dependency on Collections.Sequences.